src/components/orgunits/SelectedOrgUnits.js

```diff
@@ -16,32 +16,32 @@ const styles = {
     },
 };
 
-const levels = {
+const getLevels = () => ({
     USER_ORGUNIT: i18n.t('user organisation unit'),
     USER_ORGUNIT_CHILDREN: i18n.t('right below user organisation unit'),
     USER_ORGUNIT_GRANDCHILDREN: i18n.t(
         'two levels below user organisation unit'
     ),
-};
+});
 
-const modes = {
+const getModes = () => ({
     SELECTED: i18n.t('in'),
```

Comment:
I agree these are hard to translate without the context, I've changed to:

```
const getModeString = props => ({
    SELECTED: i18n.t('{{units}} in {{orgunits}}', props),
    CHILDREN: i18n.t('{{units}} in and right below {{orgunits}}', props),
    DESCENDANTS: i18n.t('{{units}} in and all below {{orgunits}}', props),
});
```

Reply:
Great success! Tiny note, I would prefer `const getModeString = (units, orgunits) => ({` to avoid having to pre-construct the props object like so: https://github.com/dhis2/maps-app/blob/4bdf785c65b9bfa65690b09559ea8d089e2dae9c/src/components/orgunits/SelectedOrgUnits.js#L44. But I guess this is still readable enough 👍